src/acceptance/helpers/helpers.go

```diff
@@ -377,13 +377,13 @@ func RunningInstances(appGUID string, timeout time.Duration) int {
 	return webInstances
 }
 
-func WaitForNInstancesRunning(appGUID string, instances int, timeout time.Duration) {
+func WaitForNInstancesRunning(appGUID string, instances int, timeout time.Duration, optionalDescription ...interface{}) {
```

Comment:
```suggestion
func WaitForNInstancesRunning(appGUID string, instances int, timeout time.Duration, format string, a ...any) {
```

Reply:
If you use the above then you do force a description of the assertion and IntelliJ will check the that the format items are supplied correctly